build/common.gni

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains common build flags for building Crosswalk. Users should
+# NOT directly import this file into their build folder.
+# See //xwalk/build/linux.gni for how to import flags for Crosswalk-Linux Build
+
+# Crosswalk specific build flags are defined in xwalk.gni. Import them.
+import("//xwalk/build/xwalk.gni")
+
+root_extra_deps = [ "//xwalk:xwalk_builder" ]
+
+# From src/build/config/features.gni
+enable_nacl = true
+
+# From src/build/config/features.gni
+enable_webvr = false
+
+# From src/build/config/compiler/BUILD.gn
+# If we want stack unwind support for backtrace()
+exclude_unwind_tables = false
```

Comment:
Ooops, I hate it when they change the meaning of a variable like that :-)

Still, the effect is the same:
* in gyp `release_unwind_tables` is on by default, and disabled for official Chrome builds (and also on Android for us)
* in gn, `exclude_unwind_tables` is off by default, enabled for official Chrome builds and we want it to be enabled on Android (i.e. you only want to change this value to `true` in a future `android.gni`).